### PR TITLE
feature/#7 Verifying if answer is correct

### LINKED ISSUE #7: Logic | Verifying if answer is correct
old hash: #23

### PR DESCRIPTION
**Pull request by paula0403**
Closes #7

### CODE REVIEW

File: src/app/components/VerifyingIfAnswerIsCorrect.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+export default function VerifyingAnswerIsCorrect() {
+  const userAnswerIsCorrect = { correct: "" };
+  function verifyingAnswer(userAnswer, rightAnswer) {
+    userAnswer === rightAnswer ? userAnswerIsCorrect.correct = true : userAnswerIsCorrect.correct = false;
+
+    console.log(userAnswerIsCorrect);
```
Comment (synowa) [inline src/app/components/VerifyingIfAnswerIsCorrect.js:7]: It's not supposed to be in the production code.

File: src/app/components/VerifyingIfAnswerIsCorrect.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    return userAnswerIsCorrect
```
Comment (synowa) [inline src/app/components/VerifyingIfAnswerIsCorrect.js:8]: As we talked, you can consider changing return type to just boolean.

File: src/app/App.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import createWhiteButtonWithIcon from './components/ButtonWhiteWithIcon';
+import VerifyingAnswerIsCorrect from './Components/VerifyingIfAnswerIsCorrect';
```
Comment (synowa) [inline src/app/App.js:2]: I think this import is not needed unless the module is actually used.

File: src/app/components/VerifyingIfAnswerIsCorrect.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+export default function VerifyingAnswerIsCorrect() {
```
Comment (synowa) [inline src/app/components/VerifyingIfAnswerIsCorrect.js:2]: I think you should export only one function. Right now if you import VerifyingAnswerIsCorrect(), there is no way to call verifyingAnswer.

**Review by synowa — CHANGES_REQUESTED** [review]

File: src/app/components/VerifyingIfAnswerIsCorrect.js (hunk 1 above)
Comment (paula0403) [reply src/app/components/VerifyingIfAnswerIsCorrect.js]: It was removed

File: src/app/App.js (hunk 3 above)
Comment (paula0403) [reply src/app/App.js]: Now is without import.

File: src/app/components/VerifyingIfAnswerIsCorrect.js (hunk 4 above)
Comment (paula0403) [reply src/app/components/VerifyingIfAnswerIsCorrect.js]: I changed function, now is only one.

File: src/app/App.js [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import createWhiteButtonWithIcon from './components/ButtonWhiteWithIcon';
 
+
```
Comment (synowa) [inline src/app/App.js:3]: It looks like new lines in this file are accidental. Could you remove or revert changes to this file?

**Review by synowa — APPROVED** [review]

File: src/app/App.js [hunk 6]
```diff
@@ -8,6 +9,8 @@ const App = ({ options }) => {
     'fa-graduation-cap',
   );
   quiz.append(buttonRules);
+  ;
```
Comment (DominikNowak) [inline src/app/App.js:12]: What's for that semicolon here?

File: src/app/components/VerifyingIfAnswerIsCorrect.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+
+export default function VerifyingAnswerIsCorrect(userAnswer, rightAnswer) {
+  const userAnswerIsCorrect = { correct: "" };
+  userAnswer === rightAnswer ? userAnswerIsCorrect.correct = true : userAnswerIsCorrect.correct = false;
+  return userAnswerIsCorrect
```
Comment (DominikNowak) [inline src/app/components/VerifyingIfAnswerIsCorrect.js:5]: Here should be semicolon

File: src/app/components/VerifyingIfAnswerIsCorrect.js [hunk 8]
```diff
@@ -0,0 +1,6 @@
+
+export default function VerifyingAnswerIsCorrect(userAnswer, rightAnswer) {
```
Comment (DominikNowak) [inline src/app/components/VerifyingIfAnswerIsCorrect.js:2]: I think this name is overcomplicated. What you think about "AnswerVerification" or "IsAnswerCorrect"?

File: src/app/components/VerifyingIfAnswerIsCorrect.js (hunk 7 above)
Comment (DominikNowak) [inline src/app/components/VerifyingIfAnswerIsCorrect.js:5]: Does it have to return Object? 
If it's only have to compare 2 answers, why not write it just like this:
`return userAnswer===rightAnswer;`

**Review by DominikNowak — CHANGES_REQUESTED** [review]
I don't see any linked issue, title says nothing to me, so honestly I don't even know what's going on here, whether the requirements are met or not. Also please use format script before you commit and push changes, certainly before PR.

File: src/app/components/VerifyingIfAnswerIsCorrect.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (DominikNowak) [inline src/app/components/VerifyingIfAnswerIsCorrect.js:1]: Shouldn't be empty line here

Comment (DominikNowak) [reply src/app/components/VerifyingIfAnswerIsCorrect.js:1]: and it's not component, should be in utils, file name from small letter.

File: src/app/components/VerifyingIfAnswerIsCorrect.js (hunk 8 above)
Comment (paula0403) [reply src/app/components/VerifyingIfAnswerIsCorrect.js:2]: You're right, the shorter name will be better and cleaner.

File: src/app/App.js (hunk 6 above)
Comment (paula0403) [reply src/app/App.js:12]: Removed

**Review by synowa — APPROVED** [review]
Thanks for changes, looks good now.

**Review by DominikNowak — APPROVED** [review]
it's ok

**Pull request merged**